usim/py/resources/container.py

```diff
@@ -0,0 +1,110 @@
+from .base import BaseResource, Put, Get
+
+
+class ContainerPut(Put):
+    """Request to put ``amount`` of resources into the ``container``"""
+    def __init__(self, container: 'Container', amount: float):
+        if amount <= 0:
+            raise ValueError('amount must be greater than 0')
+        self.amount = amount
+        super(ContainerPut, self).__init__(container)
+
+
+class ContainerGet(Get):
+    """Request to get ``amount`` of resources out of the ``container``"""
+    def __init__(self, container, amount):
+        if amount <= 0:
+            raise ValueError('amount must be greater than 0')
+        self.amount = amount
+        super(ContainerGet, self).__init__(container)
+
+
+class Container(BaseResource):
+    """
+    Resource with ``capacity`` of continuous, indistinguishable content
+
+    A Process may :py:meth:`~.get` out or :py:meth:`~.put` in an arbitrary
+    amount of content, provided there is content or ``capacity`` available.
+    Requests that cannot be granted immediately are served once sufficient
+    content or capacity are available.
+
+    :param capacity: the maximum amount of content available
+    :param init: the initial amount of content available
+
+    By default, a :py:class:`~.Container` has infinite capacity but
+    starts empty. Note that it is not possible to have a higher initial
+    content than maximum capacity.
+
+    .. hint::
+
+        **Migrating to μSim**
+
+        The closest equivalent of a :py:class:`~.Container` in μSim
+        are :py:class:`~usim.basics.Resources`. Each contains *multiple*
+        resource capacities which can be increased, decreased, set, borrowed
+        or claimed individually or together.
+        To emulate a :py:class:`~.Container`, use :py:class:`~usim.basics.Resources`
+        with a single resource type:
+
+        .. code:: python3
+
+            resources = Resources(capacity=8)
+            # put more capacity into the resource
+            await resource.increase(capacity=8)
+            # get some capacity from the resource
+            await resource.decrease(capacity=8)
```

Comment:
This code might lead to confusion. After reading the documentation above, you distinguish `capacity` and `init`. This example uses `capacity` only. So it might be unclear which `capacity` this resource does have, what is the actual maximum capacity and the available resources. It might help to put a short comment about available resources?

Reply:
I've changed this to use ``level`` instead of ``capacity``. Please take a look whether that makes it clearer.